app/views/dashboards/following_tags.html.erb

```diff
@@ -26,14 +26,21 @@
             <% if tag %>
               <% color = HexComparer.new([tag.bg_color_hex || "#0000000", tag.text_color_hex || "#ffffff"]).brightness(0.8) %>
               <div class="crayons-card branded-2 p-4 m:p-6 m:pt-4 flex flex-col single-article break-word content-center <% if follow.points < 0 %>opacity-75<% end %>" style="border-top-color: <%= color %>;" id="follows-<%= follow.id %>">
-                <h3 class="s:mb-1 -ml-1 p-0 fw-medium">
-                  <a href="/t/<%= tag.name %>" class="crayons-tag crayons-tag--l">
-                    <span class="crayons-tag__prefix">#</span><%= tag.name %>
-                  </a>
-                  <% if follow.points < 0 %>
-                    <span class="crayons-indicator crayons-indicator--critical crayons-indicator--outlined" title="This tag has negative follow weight">Anti-follow</span>
+                <div class="crayons-card__follow">
+                  <h3 class="s:mb-1 -ml-1 p-0 fw-medium">
+                    <a href="/t/<%= tag.name %>" class="crayons-tag crayons-tag--l">
+                      <span class="crayons-tag__prefix">#</span><%= tag.name %>
+                    </a>
+                    <% if follow.points < 0 %>
+                      <span class="crayons-indicator crayons-indicator--critical crayons-indicator--outlined" title="This tag has negative follow weight">Anti-follow</span>
+                    <% end %>
+                  </h3>
+                  <%= form_for(follow, html: { method: :delete }) do %>
+                    <button type="submit" class="crayons-btn crayons-btn--ghost crayons-btn--s">
+                      <%= inline_svg_tag("cancel.svg", class: "crayons-icon", aria: true, title: "Unfollow", height: 16, width: 16) %>
+                    </button>
```

Comment:
I'd recommend using x.svg instead of cancel.svg - the SVG code is just a bit cleaner.

As for button, it must include some extra class indicating it's an icon only type of button.

```suggestion
                    <button type="submit" class="crayons-btn crayons-btn--ghost crayons-btn--s crayons-btn--icon">
                      <%= inline_svg_tag("x.svg", class: "crayons-icon", aria: true, title: "Unfollow", height: 24, width: 24) %>
                    </button>
```

^ that should look slightly better.